com.unity.multiplayer.mlapi/Runtime/Messaging/CustomMessageManager.cs

```diff
@@ -32,46 +31,41 @@ public static class CustomMessagingManager
 
         internal static void InvokeUnnamedMessage(ulong clientId, Stream stream)
         {
-            if (OnUnnamedMessage != null)
-            {
-                OnUnnamedMessage(clientId, stream);
-            }
-
-            NetworkingManager.Singleton.InvokeOnIncomingCustomMessage(clientId, stream);
+            OnUnnamedMessage?.Invoke(clientId, stream);
```

Comment:
I'm probably missing some c# trick here, but doesn't this still need to be `InvokeOnIncomingCustomMessage`?

Reply:
`OnIncomingCustomMessage` was obsolete, now removed from the API so does `InvokeOnIncomingCustomMessage` method and call-sites.